multi_import/importer.py

```diff
@@ -398,6 +447,48 @@ def process_row(self, row, data, context):
 
         data.processed = True
 
+    def process_row_subsequent_pass(self, row, data, context, serializer_class):
+        if row.errors:
+            return
+
+        if not data.instance:
+            return
+
+        serializer = serializer_class(
+            instance=data.instance,
+            data=row.data.copy(),
+            context=context,
+            partial=data.instance is not None,
+        )
+
+        if not serializers.might_have_changes(serializer):
+            data.add_diff(serializers.get_diff_data(serializer, no_changes=True))
+            return
+
+        is_valid = serializer.is_valid()
+
+        if not is_valid:
+            row.set_errors(serializer.errors)
+            return
+
+        has_changes = serializers.has_changes(serializer)
+
+        if not has_changes:
```

Comment:
can just do `if not serializers.has_changes(serializer)` here instead of assigning above-- the variable doesn't seem to be used elsewhere.